### ISSUE TITLE
Does secucheck support inter-procedure analysis?

### ISSUE THREAD

**Issue by dyd1024** [secure-software-engineering/secucheck#7]
Hello, I want to confirm whether secucheck  supports interprocedural analysis? Thanks!

**Comment by ranjithkris** [comment]
> Hello, I want to confirm whether secucheck  supports interprocedural analysis? Thanks!

Yes, SecuCheck performs interprocedural analysis. We also tested this by keeping source and sink in different functions. You can check those examples in Secucheck catalog: https://github.com/secure-software-engineering/secucheck-catalog/tree/v1.2.0
